zipline/pipeline/loaders/blaze/core.py

```diff
@@ -169,16 +167,17 @@
     normalize_data_query_bounds,
     normalize_timestamp_to_query_time,
 )
-from zipline.pipeline.term import NotSpecified
-from zipline.lib.adjusted_array import AdjustedArray
+from zipline.pipeline.term import NotSpecified, NotSpecifiedType
+from zipline.lib.adjusted_array import AdjustedArray, can_represent_dtype
 from zipline.lib.adjustment import Float64Overwrite
 from zipline.utils.enum import enum
 from zipline.utils.input_validation import (
     expect_element,
     ensure_timezone,
     optionally,
 )
-from zipline.utils.numpy_utils import repeat_last_axis
+from zipline.utils.numpy_utils import categorical_dtype, repeat_last_axis, \
```

Comment:
Style nit: we generally prefer wrapping imports with parens over line breaks elsewhere in zipline.